packages/io/package.json

```diff
@@ -43,8 +43,11 @@
     "rxjs": "7.8.1"
   },
   "devDependencies": {
+    "@ckb-lumos/base": "0.20.0",
+    "@ckb-lumos/rpc": "0.20.0",
     "@types/koa-compose": "3.2.5",
     "@types/koa-router": "7.4.4",
-    "koa": "2.14.2"
+    "koa": "2.14.2",
+    "reflect-metadata": "0.1.13"
```

Comment:
we have added `@ckb-lumos/lumos` at 37 line, these devDependencies are unnecessary.

Reply:
They will be phantom dependencies if they are not specified in the list. And the code will break once phantom dependencies are not installed along with `@ckb-lumos/lumos`

Reply:
`@ckb-lumos/lumos` has added all sub-packages of the Lumos to dependencies. When the sub-packages will not install?

Reply:
> `@ckb-lumos/lumos` has added all sub-packages of the Lumos to dependencies. When the sub-packages will not install?

Future is hard to predicate 😑 here're 3 cases for the explanation.

1. A sub-package is removed from `@ckb-lumos/lumos`: It will fail to import the sub-package because it's removed from `@/ckb-lumos/lumos` and won't be installed implicitly
2. Incompatibility introduced by implicitly installed sub-package. Using an implicitly installed package is similar to using an unexported method of a package. It's an internal implementation and is not intended for public usage. The compatibility won't be guaranteed by the maintainer. Say a method named `A` is imported from a `phantom package@v0.0.1`, but it's removed since `phantom package@v0.0.2`, the code breaks because `phantom package` is not explicitly specified in the package.json.
3. If `phantom dependency` is acceptable, will you skip installing `lodash` if `lumos` uses it?